### ISSUE TITLE
New translation: Slovenian - SI

### ISSUE THREAD

**Issue by SloCompTech** [squidfunk/mkdocs-material#1135]
Hi,
here are translations for Slovenian language.

``` jinja
{% macro t(key) %}{{ {
  "language": "si",
  "clipboard.copy": "Kopiraj v odložišče",
  "clipboard.copied": "Kopirano v odložišče",
  "edit.link.title": "Uredi stran",
  "footer.previous": "Prejšnja stran",
  "footer.next": "Naslednja stran",
  "meta.comments": "Komentarji",
  "meta.source": "Izvorna koda",
  "search.language": "si",
  "search.placeholder": "Išči",
  "search.result.placeholder": "Vpiši iskalni niz",
  "search.result.none": "Ni zadetkov",
  "search.result.one": "1 zadetek",
  "search.result.other": "# zadetkov",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "Skoči na vsebino",
  "source.link.title": "Pojdi na repozitorij",
  "toc.title": "Kazalo"
}[key] }}{% endmacro %}
```

Thanks!

**Comment by squidfunk** [comment]
Awesome, thanks!

**Comment by squidfunk** [comment]
Released as part of 4.4.0.
